src/platform/plugins/shared/data/public/search/session/sessions_mgmt/components/table/utils/map_to_ui_session.ts

```diff
@@ -66,7 +67,7 @@ export const mapToUISession = ({
     locatorId,
     enableOpeningInNewTab
       ? {
-          ...restoreState,
+          ...omit(restoreState, 'savedSearchId'),
```

Comment:
Tabs won't be cleared if opened a search session associated with another discover session id